src/uni_modules/wot-ui/components/wd-color-picker/index.scss

```
@@ -0,0 +1,367 @@
+    height: $color-picker-thumb-size;
+    border: $color-picker-thumb-border;
+    border-radius: $radius-full;
+    box-shadow: $color-picker-thumb-shadow;
+    box-sizing: border-box;
+    transform: translate(-50%, -50%);
+  }
+
+  @include e(slider) {
+    margin-top: $color-picker-slider-margin-top;
+
+    &.wd-slider--horizontal {
```

Stylelint (17.14.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)